vertx-circuit-breaker/src/main/java/io/vertx/ext/circuitbreaker/CircuitBreakerState.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2011-2016 The original author or authors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Apache License v2.0 which accompanies this distribution.
+ *
+ *      The Eclipse Public License is available at
+ *      http://www.eclipse.org/legal/epl-v10.html
+ *
+ *      The Apache License v2.0 is available at
+ *      http://www.opensource.org/licenses/apache2.0.php
+ *
+ * You may elect to redistribute this code under either of these licenses.
+ */
+
+package io.vertx.ext.circuitbreaker;
+
+/**
+ * Circuit breaker states.
+ *
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ */
+public enum CircuitBreakerState {
+  OPEN,
```

Comment:
document the enum values, as it generates .adoc and it is going to be present in reference doc